HW4/task3.py

```diff
@@ -0,0 +1,14 @@
+# Напишите функцию принимающую на вход только ключевые параметры и возвращающую словарь,
+# где ключ — значение переданного аргумента, а значение — имя аргумента.
+# Если ключ не хешируем, используйте его строковое представление.
+#
+
+def get_dict(first=None, second=None, third=None):
+    result = {}
+    for k, v in locals().items():
+        if v and v.__hash__:
+            result[v] = k
+    return result
```

Comment:
По условию, если ключ нехэшируемый, то нужно взять его строковое представление.
```suggestion
        if not v.__hash__:
            v = str(v)
        result[v] = k
        
    return result
```